src/modules/api/pipeline-task/pipelines/utils.py

```diff
@@ -95,7 +95,7 @@ def start_job(payload, task_route, run_if_exists=False):
   except HttpError as ex:
     if ex.status_code == 400:
       time.sleep(5)
-      run_response = handler.run_job()
+      run_response, pub_sub_id = handler.run_job()
```

Comment:
thanks for the confirmation. An alternative is to use a `retry` library/decorator where you can specify the # of retries, wait time, backoff scale, etc.